plugins/Quotes/class.quotes.plugin.php

```diff
@@ -544,6 +541,20 @@ protected function formatQuote($type, $id, &$quoteData, $format = false) {
         Emoji::instance()->enabled = $emojiEnabled;
     }
 
+    /**
+     * Checks if the user can view discussion.
+     *
+     * @param object $discussion
+     * @throws Exception If the user cannot view the discussion.
+     */
+    private function canViewDiscussion(object $discussion): void {
```

Comment:
This is more of a nitpick, but naming like this implies there will be a boolean return value. It's like you're asking a question: "Can this user view the discussion, yes or no?" and the answer would be the return value. This is reinforced by `DiscussionModel`'s own `canView` method, which returns a value. It might make more sense to phrase this in the imperative mood, like `verifyDiscussionViewPermissions` or something.

Reply:
Do you think some form of this method would be beneficial to add to `DiscussionModel`?

Reply:
I've already used it twice. It would be beneficial.